bin/gh_pr_bootstrap.sh

```diff
@@ -92,7 +92,7 @@ function setup_cmsbot() {
     if [ ! -d ${CMS_BOT_DIR} ]; then
         (
             cd "$WORKSPACE"
-            git clone -b master git@github.com:Mu2e/CI
+            git clone -b main git@github.com:Mu2e/CI
```

Comment:
```suggestion
            git clone git@github.com:Mu2e/CI
```

This would be ideal I think